translations/translation_utility.sh

```diff
@@ -65,6 +66,14 @@ Check_bin() { # Check if the bin path exists
         export PATH="$PATH":$bin_path # Set temporary path
     fi
 }
+
+OpenLinguist() {
+    cd "$bin_path" || return
+    echo -e "Opening Linguist"
+    open -a linguist.app "$1" || return
```

Comment:
Yep. that's be Mac specifig. it's be `linguist.exe` on Windows and jist `linguist` on Linux. But at least on Windows just 'linguist' should work too, not sure whether it would on Mac also?

Reply:
`$@` here, to allow for more than one filename